Sources/GeneratorCLI/GeneratorCLI.swift

```diff
@@ -222,6 +223,56 @@ extension GeneratorCLI {
       return false
     }
   }
+
+  struct MakeWasmSDK: AsyncParsableCommand {
+    static let configuration = CommandConfiguration(
+      commandName: "make-wasm-sdk",
+      abstract: "Experimental: Generate a Swift SDK bundle for WebAssembly.",
+      discussion: """
+      The default `--target` triple is wasm32-unknown-wasi
+      """
+    )
+
+    @OptionGroup
+    var generatorOptions: GeneratorOptions
+
+    @Option(
+      help: """
+      Path to the Swift toolchain package containing the Swift compiler that runs on the host platform.
+      """
+    )
+    var hostSwiftPackagePath: String
```

Comment:
Should `--host-swift-package-path` and `--target-swift-package-path` be common options between different SDK generators?   The Linux generator currently assumes `Artifacts` and `Bundles`, but it also downloads the archives and unpacks them.

Reply:
Yes, they could be common options. Let me open a separate PR for it.